File: GTTS/INL.VettingService.Models/RemovedParticipant_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.VettingService.Models
{
    public class RemovedParticipant_Item : IRemovedParticipant_Item
    {
        public long PersonID { get; set; }
        public bool RemovedFromEvent { get; set; }
        public bool RemovedFromVetting { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.VettingService.Models/removed-participant_item.ts


import { IRemovedParticipant_Item } from './iremoved-participant_item';

export class RemovedParticipant_Item implements IRemovedParticipant_Item {
  
	public PersonID: number = 0;
	public RemovedFromEvent: boolean = false;
	public RemovedFromVetting: boolean = false;
  
}


